cosmwasm/packages/sgx-vm/src/attestation.rs

```diff
@@ -127,7 +127,7 @@ pub extern "C" fn ocall_get_update_info(
 }
 
 pub fn create_attestation_report_u(spid: &[u8], api_key: &[u8]) -> SgxResult<()> {
-    let enclave = get_enclave()?;
+    let enclave = get_enclave()?.ok_or(sgx_status_t::SGX_ERROR_OUT_OF_TCS)?;
 
```

Comment:
when `get_enclave` returns `Ok(None)` that means that the enclave is busy. In most code contexts, such as this one, it's in calls to the enclave while the node is still being configured. We don't expect this to really happen but because i preferred to not panic, i just convert it to the original error.